README.md

```diff
@@ -76,3 +76,48 @@ Solution:
 3. Change related ip of host.docker.internal and gateway.docker.internal to your new ip (found in terminal using the ipconfig command: e.g. 192.168.0.1)
 4. save
 5. restart the application.
+
+## Adding an ADR Algorithm
+When the ADR Algorithm has been tested, and is ready for deployment, it has to be built as a Go Module.
+
+The module is created running:
+```bash
+mkdir example-mod-name`.
```

Comment:
Is the backtick at the end of the line a mistake?
Same comment to the line below

Reply:
You are right, the backticks has been remove on both lines